uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/option-group-mock-data.js

```diff
@@ -0,0 +1,328 @@
+define([
```

Comment:
It would keep things more manageable if instead of having a single, huge "option-group" file, you had smaller, individual files for each option group. A file for the day types, one for the leave request statuses, and so on.